doc/arkode/guide/source/Constants.rst

```diff
@@ -74,249 +74,6 @@ contains the ARKODE output constants.
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKODE_ARK437L2SA_ERK_7_3_4`          | Use the ARK-7-3-4 ERK method.                              |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKODE_ARK548L2SAb_ERK_8_4_5`         | Use the ARK-8-4-5b ERK method.                             |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKODE_SHU_OSHER_3_2_3`               | Use the Shu-Osher-3-2-3 ERK method.                        |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKODE_VERNER_9_5_6`                  | Use the Verner-9-5-6 ERK method.                           |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKODE_VERNER_10_6_7`                 | Use the Verner-10-6-7 ERK method.                          |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKODE_VERNER_13_7_8`                 | Use the Verner-13-7-8 ERK method.                          |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKODE_VERNER_16_8_9`                 | Use the Verner-16-8-9 ERK method.                          |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKSTEP_DEFAULT_ERK_1`                | Use ARKStep's default first-order ERK method               |
-   |                                               | (ARKODE_FORWARD_EULER_1_1).                                |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKSTEP_DEFAULT_ERK_2`                | Use ARKStep's default second-order ERK method              |
-   |                                               | (ARKODE_HEUN_EULER_2_1_2).                                 |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKSTEP_DEFAULT_ERK_3`                | Use ARKStep's default third-order ERK method               |
-   |                                               | (ARKODE_BOGACKI_SHAMPINE_4_2_3).                           |
-   +-----------------------------------------------+------------------------------------------------------------+
-   | :index:`ARKSTEP_DEFAULT_ERK_4`                | Use ARKStep's default fourth-order ERK method              |
-   |                                               | (ARKODE_ZONNEVELD_5_3_4).                                  |
```

Comment:
While I really like having a table with method properties in `Butcher.rst`, the documentation now omits any reference to the constants for "default" methods, e.g., `ARKSTEP_DEFAULT_ERK_2` and `ARKSTEP_DEFAULT_ARK_ETABLE_5`.  I wonder if those constants could be retained in the tables on this page?

Reply:
The new table denotes the default method with an asterisk in the order column. Maybe this could be made clearer by putting "(default)" after the name in the new table.

Reply:
Sure, but in the current form of this PR, users are never informed of the "names" `ARKSTEP_DEFAULT_ERK_2`, etc.  Perhaps we don't care for users to know about those names since they're already the defaults?  But if we could anticipate a scenario where a user would want to specify those names/constants, then we should let them know what they are.

Reply:
Ah, I see what you mean. Yes, it would be good to keep the `ARKSTEP_DEFAULT_*` entries in this table or have a separate table with just the defaults.

Reply:
I added them back to the table in `Constants.rst`